### PR TITLE
fix: Optimize project pipeline customfilter

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
fix: Optimize project pipeline customfilter

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb24iOls4ODMsNzcyXSwibWVtYmVyIjpbIjEwMDEyNjEiXX0%3D&id=275813&iterationID=772&pId=0&type=BUG)

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       Optimize project pipeline customfilter       |
| 🇨🇳 中文    |     项目级流水线过滤交互优化         |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ func (p *CustomFilter) ConditionRetriever() ([]interface{}, error) {
 		return nil, err
 	}
 	creatorCondition.ConditionBase.Placeholder = cputil.I18n(p.sdk.Ctx, "please-choose-creator")
+	creatorCondition.ConditionBase.Disabled = func() bool {
```
Comment (sfwn) [inline modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go:47]: no need function wrapper

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go (hunk 1 above)
Comment (littlejiancc) [reply modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**